app/src/main/java/org/simple/clinic/patient/PatientRepository.kt

```diff
@@ -68,42 +57,12 @@ class PatientRepository @Inject constructor(
     }
   }
 
-  private fun searchByName(patientName: String, facilityId: UUID): List<PatientSearchResult> {
-    return reportTimeTaken(
-        clock = utcClock,
-        operation = "Instant Search Patient:Loading Search Result for Facility: $facilityId") {
-      database.patientSearchDao().searchByName(patientName, facilityId)
-    }
-  }
-
   private fun searchByNamePagingSource(patientName: String, facilityId: UUID): PagingSource<Int, PatientSearchResult> {
     return database
         .patientSearchDao()
         .searchByNamePagingSource(patientName, facilityId)
   }
 
-  private fun searchByPhoneNumber(
-      phoneNumber: String,
-      facilityId: UUID
-  ): List<PatientSearchResult> {
-    return reportTimeTaken(
```

Comment:
Can the `reportTimeTaken` method be removed now?

Reply:
Are we not using that anywhere else apart from `PatinetRepository`? If not, then yeah. We can remove the `reportTimeTaken` method from the project. Will open a different PR for it.

Reply:
Yeah, I see in your branch that it is unused. We can remove this and the corresponding method in `Analytics` as well.